### DISCUSSION TITLE
License is for Firefox but extension build is for Chrome - Opera GX

### DISCUSSION THREAD

**Discussion by chrosion** [aclap-dev/video-downloadhelper#180] [License & Purchase requests]
### What's your issue?

I installed the Chrome browser extension on Opera GX and bought the license there too. Now it says "License is for Firefox but extension build is for Chrome" and I can't use it on Opera GX. Is there a way to convert it to Chrome? I have no idea why it bought a Firefox one

EDIT: I ended up buying the Chrome one. Now I just want a refund for the Firefox license. Is this possible?

### The 10 first digits (only 10) of your license number or email address

8cb8a65a70

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
